command/repo.go

```diff
@@ -2,19 +2,30 @@ package command
 
 import (
 	"fmt"
+	"net/url"
 	"os"
 	"strings"
+	"time"
 
+	"github.com/cli/cli/api"
 	"github.com/cli/cli/git"
 	"github.com/cli/cli/internal/ghrepo"
 	"github.com/cli/cli/utils"
+
+	"github.com/AlecAivazis/survey/v2"
 	"github.com/spf13/cobra"
 )
 
 func init() {
 	RootCmd.AddCommand(repoCmd)
 	repoCmd.AddCommand(repoCloneCmd)
 	repoCmd.AddCommand(repoViewCmd)
+	repoCmd.AddCommand(repoForkCmd)
+
+	repoForkCmd.Flags().StringP("clone", "c", "prompt", "Clone fork. Pass false to not clone fork.")
```

Comment:
1. Could we make this a boolean flag, respect the true/false setting if explicitly specified, and default to prompting if the flag wasn't passed?
2. Do we need to have a shorthand for this? I imagine that people will mostly use these flags from scripts, where it's generally beneficial for flags to be spelled out in full (e.g. `--clone` vs. `-c`) and therefore we could consider not providing shorthand options at all to force that explicitness. Furthermore, it's only possible to pass `--clone=false` for boolean flags with long-form flags I think (I'm not sure if `-c=false` would work, have to check)

Reply:
> (I do not know how to query for whether a flag was "seen" or not but I imagine there should be a way?)

I started with this approach and could not find (surprisingly) an easy way to tell if a flag was specified by the user or not. It's possible that switching back to BoolVar would help (as we could get a nil pointer, I'd imagine) but I was hoping to keep consistency with the other flag definitions.

> Do we need to have a shorthand for this?

I guess not. `-cfalse` works but I agree that these are largely for scripting and can be explicit/long form/